serde-api/gradle.properties

```diff
@@ -0,0 +1,2 @@
+sonatypeUsername=test
```

Comment:
do we need to commit these?

Reply:
Yes, we need any value to be present during build time

Reply:
Can any value be ""? `test` is a bit confusing

Reply:
I agree that it's a bit confusing